CHANGES.md

```diff
@@ -16,6 +16,13 @@
 * The `lrlex::build_lex` function has been deprecated in favour of
   `LRNonStreamingLexerDef::from_str`.
 
+## Bug fixes
+
+* The statetable and other elements that were previously included in the user
```

Comment:
remove "that" before "were".

Reply:
Fixed in https://github.com/softdevteam/grmtools/pull/189/commits/85117e7c1b47ff3bf2c1cd1b1028f4360fbd0ddd.